src/main/java/com/smartbear/collaborator/issue/IssueRest.java

```diff
@@ -739,10 +762,10 @@ public String createReview(CollabUserInfo collabUserInfo) throws Exception {
 			createReviewCommand.getArgs().put("title", issue.getSummary() != null ? issue.getSummary() : "");
 			createReviewCommand.getArgs().put("customFields",
 					new com.smartbear.collaborator.json.collab.CustomField[] { 
-					new com.smartbear.collaborator.json.collab.CustomField("Overview", new HashSet(Arrays.asList(issue.getDescription() != null ? issue.getDescription() : "")) )	});
-			createReviewCommand.getArgs().put("internalCustomFields",
-					new com.smartbear.collaborator.json.collab.CustomField[] { 
-					new com.smartbear.collaborator.json.collab.CustomField("JiraIssueId",  new HashSet(Arrays.asList(issue.getId().toString())) )});
+							new com.smartbear.collaborator.json.collab.CustomField("Overview", new HashSet(Arrays.asList(issue.getDescription() != null ? issue.getDescription() : "")) ),

```

Comment:
Internal fields can't be visible. Internal field "JiraIssueId" exists by default since Collaborator 9.2
Internal custom fields are only for internal use for Collaborator, so there is no access to such fields from Collaborator.
The only way I see is you can patch code and add issue id to internal custom field and to regular custom field as you did before.